utils/workspace/uncertainties.py

```diff
@@ -35,6 +39,21 @@ def get_lumi_unc(year: str, analysis: str) -> dict[str, Any]:
     }[year]
 
 
+def get_prefiring_shape(year: str, analysis: str) -> dict[str, Any]:
+    """Return prefiring shape systematics for a given year and analysis."""
+    proc_list = get_processes_by_type(analysis=analysis)
```

Comment:
Agreed yesterday to have this only on minor backgrounds and signals.

Can you make the same change also for pileup?